### PR TITLE
dufs: Add version 0.30.0

### LINKED ISSUE #3898: [Request] dufs: a mini http web server
### Criteria

For a package to be acceptable in the Main bucket, it should be:

- [x] a non-GUI tool
- [x] a reasonably well-known and widely used developer tool (e.g. if it's a GitHub project, it should have at least 500 stars and 150 forks)
- [x] the latest stable version of the program
- [x] the full version i.e. not a trial version
- [x] a fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Information

**Name:** dufs

**Description:** a mini http server with upload and webdav support

**Homepage:** https://github.com/sigoden/dufs

**Download link(s):** https://github.com/sigoden/dufs/releases

**Some indication of popularity/repute:** 1.5K

### PR DESCRIPTION
**Pull request by ViCrack**
Closes #3898

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
`bin` can be moved outside architecture

**Comment by ViCrack** [comment]
I have moved

File: bucket/dufs.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "0.30.0",
+    "description": "A file server that supports static serving, uploading, searching, accessing control, webdav...",
+    "homepage": "https://github.com/sigoden/dufs",
+    "license": "Freeware",
```
Comment (rashil2000) [inline bucket/dufs.json:5]:
```suggestion
    "license": "MIT|Apache",
```

**Pull request merged**